Include/Acidanthera/Library/OcAppleEventLib.h

```diff
@@ -18,6 +18,9 @@
 
 #include <Protocol/AppleEvent.h>
 
+#define POINTER_POLL_DEFAULT_FREQUENCY       0
```

Comment:
Not as sure about this comment, but possibly the code is more readable if this is called `POINTER_POLL_DEFAULT`? With the current name then where used in the code, I initially read it as containing the default frequency value to use, rather than as being a placeholder which will be replaced with the default frequency.